pandas/util/testing.py

```diff
@@ -1185,8 +1201,11 @@ def assert_series_equal(
                 right._internal_get_values(),
                 check_dtype=check_dtype,
             )
-    elif is_interval_dtype(left) or is_interval_dtype(right):
-        assert_interval_array_equal(left.array, right.array)
+    elif is_interval_dtype(left) or is_interval_dtype(left):
```

Comment:
this is now just `elif is_interval_dtype(left)` or should the second condition not have been changed?

Reply:
Good catch, I think change was my mistake. Have removed it.